.github/workflows/build-container.yml

```diff
@@ -0,0 +1,61 @@
+name: Release ingress-netperf container image
+
+on:
+  pull_request:
+    paths:
+      - 'containers/**' # Trigger only when Dockerfile changes in a pull request
+
+env:
+  CONTAINER_REGISTRY: ${{ github.repository_owner == 'cloud-bulldozer' && 'quay.io' }}
+  GO_VER: 1.19
+
+jobs:
+
+  build-and-push-container-images:
+    name: Build and Push Container Images
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+    
+    steps:
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Setup docker-buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          buildkitd-flags: --debug
+          config-inline: |
+            [worker.oci]
+              max-parallelism = 1
```

Comment:
I think we don't need this step

Reply:
If we are using Podman altogether this is not required.
That made me re-think this [here](https://github.com/cloud-bulldozer/ingress-perf/pull/34#discussion_r1330401208)

Reply:
Not needed, updated this code.